tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -747,161 +747,97 @@ def test_mpu_after_ctrl_pod_failover(self):
-                                  len(event_di_bkt), "Expected pass in read and di check "
-                                                     "operations. Found failures in READ: "
-                                                     f"{fail_bkt_get} {event_bkt_get}"
-                                                     f"or DI_CHECK: {fail_di_bkt} {event_di_bkt}")
-        LOGGER.info("Successfully performed READs on the all buckets.")
-        LOGGER.info("Step 1: Successfully create IAM user and performed WRITEs-READs-Verify with "
-                    "variable sizes objects.")
-
-        LOGGER.info("Control pod %s is hosted on %s node", self.control_pod_name, self.control_node)
-
-        failover_node = self.system_random.choice([ele for ele in self.host_worker_list if ele !=
-                                                   self.control_node])
-        LOGGER.debug("Fail over node is: %s", failover_node)
-
-        LOGGER.info("Step 2: Failover control pod %s to node %s and check cluster status",
-                    self.control_pod_name, failover_node)
-        pod_yaml = {self.control_pod_name: self.modified_yaml}
-        resp = self.ha_obj.failover_pod(pod_obj=self.node_master_list[0], pod_yaml=pod_yaml,
-                                        failover_node=failover_node)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 2: Successfully failed over control pod to %s. Cluster is in good state",
-                    failover_node)
-
-        for cnt in range(HA_CFG["common_params"]["loop_count"]):
-            LOGGER.info("Loop: %s", cnt)
-            control_pods = self.node_master_list[0].get_pods_node_fqdn(
-                const.CONTROL_POD_NAME_PREFIX)
-            control_pod_name = list(control_pods.keys())[0]
-            control_node = control_pods.get(control_pod_name)
-            LOGGER.info("Control pod %s is hosted on %s node", control_pod_name, control_node)
-
-            failover_node = self.system_random.choice([ele for ele in self.host_worker_list if
-                                                       ele != control_node])
-            LOGGER.debug("Fail over node is: %s", failover_node)
-
-            LOGGER.info("Step 3: Failover control pod %s to node %s by changing its NodeSelector "
-                        "and check cluster status. \nFailover count: %s", control_pod_name,
-                        failover_node, cnt)
-            resp = self.ha_obj.change_pod_node(self.node_master_list[0],
-                                               pod_node={control_pod_name: failover_node})
-            assert_utils.assert_true(resp[0], resp)
-            LOGGER.info("Check cluster status")
+        for count in range(num_users):
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[count],
+                                                        log_prefix=self.test_prefix,
+                                                        skipcleanup=True)
```

Comment:
If number of user is more then it will install s3bench that many times
shall we run small first and install then skip in this?

Reply:
@sonalk0209  Actually i have taken care of this, s3bench won't get installed if it is already installed.
Earlier we thought it takes latest s3bench everytime, but that's not the case.
It takes same s3bench version each time, this one:
https://github.com/Seagate/s3bench/releases/download/v2022-03-14/s3bench.2022-03-14

SO may be we can remove code from line 791 to 794.